simple.css

```diff
@@ -179,7 +179,8 @@ a:hover {
 }
 
 button,
-[role="button"],
+.button,
+a.button, /* extra specificity to override a */
```

Comment:
I removed `a.button` in my tests and didn't see a change, can you confirm?

Reply:
Yup... you're right. not sure what was going on there.